.env.sample

```diff
@@ -7,8 +7,8 @@ export DATABASE_URL=
 # export MIRROR=1
 
 # Key to sign and encrypt cookies with. Change this to a long, random string
-# for production.
-export SESSION_KEY=badkey
+# that is at least 32 bytes for production
```

Comment:
Weren't you seeing dev not able to start if this wasn't at least 32 bytes all the time? So I think this should be split up differently, so that "32 bytes" isn't connected to "for production". Something like:

```
# Key to sign and encrypt cookies with. Must be at least 32 bytes. Change this 
# to a long, random string for production.
```

?